src/world/GameCata/Handlers/CharacterHandler.cpp

```diff
@@ -115,15 +115,15 @@ void WorldSession::CharacterEnumProc(QueryResult* result)
             uint32_t char_flags = 0;
 
             if (charEnum.banned && (charEnum.banned < 10 || charEnum.banned > (uint32_t)UNIXTIME))
-                char_flags |= PLAYER_FLAG_IS_BANNED;
+                char_flags |= PLAYER_FLAG_PREVENT_MELEE_SPELLS;
             if (charEnum.deathState != 0)
-                char_flags |= PLAYER_FLAG_IS_DEAD;
+                char_flags |= PLAYER_FLAG_PLAYED_5_HOURS;
```

Comment:
If these flags are incorrect, please add a comment explaining your change.

Reply:
I tested these on wotlk and I forgot to include comments to other expansions. The flag value isn't changed, only the flag is renamed properly. But like you might've guessed, character screen does not use player flags.
**However this change has no impact ingame.** Correcting these character flags to their proper names is next on my to-do list. Although I have yet to verify and test them all.
I will include the comments from wotlk though.